docs/thin_provision.md

```diff
@@ -49,7 +49,36 @@ $ lvextend -L +15G lvmvg/thin_pool
 
 ### Configure Auto Extending of the Thin Pool (Configure Over-Provisioning protection)
 
-Editing the settings in the `/etc/lvm/lvm.conf` can allow auto growth of the thin 
+1. Since we already create thin pool as part of first thin volume provisioning,
```

Comment:
change this :
we already create thin pool  => we automatically create thin pool

Reply:
done